src/main/resources/org/sagebionetworks/web/client/widget/entity/controller/SynapseAlertViewImpl.ui.xml

```diff
@@ -1,16 +1,23 @@
 <!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
 <ui:UiBinder
   xmlns:ui="urn:ui:com.google.gwt.uibinder"
-  xmlns:g="urn:import:com.google.gwt.user.client.ui"
   xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
   xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html"
-  xmlns:e="urn:import:org.sagebionetworks.web.client.widget.entity">
-  
-  	<bh:Div>
-		<bh:Div ui:field="loginWidgetContainer" />
-  		<b:Alert ui:field="alert" type="DANGER" addStyleNames="synapseAlert margin-top-10 margin-bottom-0-imp">
-  			<bh:Div addStyleNames="iconAndMessageArea">
-	  			<b:IconStack addStyleNames="iconArea font-size-20">
+  xmlns:w="urn:import:org.sagebionetworks.web.client.widget">
+
+	<bh:Div>
+		<bh:Div ui:field="loginWidgetContainer" >
+			<w:FullWidthAlert
```

Comment:
Use a FullWidthAlert instead of the LoginWidget (which is really the LoginPage) in this case